Source/WebCore/dom/ElementInlines.h

```diff
@@ -207,9 +207,9 @@ inline const AtomString& Element::getAttribute(const QualifiedName& name, const
     return getAttribute(names...);
 }
 
-inline bool isInTopLayerOrBackdrop(const RenderStyle& style, const Element* element)
+inline bool isInTopLayerOrStyleTypeIsBackdrop(const PseudoId& styleType, const Element* element)
 {
-    return (element && element->isInTopLayer()) || style.styleType() == PseudoId::Backdrop;
+    return (element && element->isInTopLayer()) || styleType == PseudoId::Backdrop;
 }
```

Comment:
Since this is almost always used in the context of RenderObject/RenderElement, I would suggest replacing this with a method on `RenderObject` called `isTopLayerRenderer()` or `isRenderedInTopLayer()` (`::backdrop` is considered part of the top layer for rendering purposes)

This definition below should be correct (`::backdrop` is always `RenderBlockFlow` which is `RenderElement`):

```
bool RenderObject::isRenderedInTopLayer() const
{
    return false;
}
```
```
bool RenderElement::isRenderedInTopLayer() const
{
    return (element() && element()->isInTopLayer()) || (style().styleType() == PseudoId::Backdrop);
}
```